### ISSUE TITLE
another version of the English language

### ISSUE THREAD

**Issue by saol89** [usrz/bootstrap-languages#7]
Can submit a second version for the English language with this flag? Thank you :)

![Image of Yaktocat](http://vignette3.wikia.nocookie.net/potbs/images/b/b6/UK_Flag.png/revision/latest?cb=20061117162818)

**Comment by pfumagalli** [comment]
Feel free to add a pull request.

**Comment by twisk** [comment]
+1. Anyone?

**Comment by marcelstoer** [comment]
I'm neither American nor British but this is the sole reason I can't use this nice project. The closest possible origin for English is England and not the US.

**Comment by antsunji** [comment]
+1

**Comment by ntd** [comment]
![languages](https://cloud.githubusercontent.com/assets/160618/15471398/0fc3b3d4-20f6-11e6-8fc3-248809cd5ed7.png)

While working on this issue I noticed the [IconDrawer](http://www.icondrawer.com/flag-icons.php) iconset was not free enough for my purposes. I made an alternate script that regenerates the set from [linssen/country-flag-icons](https://github.com/linssen/country-flag-icons). They are scraped from Wikipedia so almost all (all?) are public domain stuff.

The default country for english has been changed to UK.

The actual code resides on a local branch (8761e4b01e453b96841a88d2510491986a30d639) in my fork: not sure it is worth a pull request or not. If this is the case just let me know.
